manual-ja.txt

```diff
@@ -240,13 +249,13 @@ AnkiDroidアイコンのボタン :: アクションバー左のアイコンを
 
 AnkiDroidでも，PC版Ankiで使うことができる検索形式をすべて使うことができ，複雑な検索に役立ちます．以下に例をいくつか示します．
 
- tag:marked :: 「マーク付き」のタグがついたカードを検索します．
+ tag:marked :: 「marked」のタグがついたカードを検索します．
  is:due :: 復習しなければいけない時期になっているカードを検索します．
  front:rabbit :: frontフィールドが「rabbit」であるカードを検索します．
 
 利用可能な検索機能の一覧は， http://wikiwiki.jp/rage2050/?2.0%2FBrowser#searching[PC版Ankiの説明書] に記載されています．
 
-また，よく使われる検索(マーク付き，保留，タグなど)については，直接入力しなくてもメニューから選んで使うこともできます．
+また，よく使われる検索(marked，保留，タグなど)については，直接入力しなくてもメニューから選んで使うこともできます．
```

Comment:
In this section, I think it is desirable that "marked" be translated to Japanese. Please revert to the strings of the old version.